lib/outputs/models.py

```diff
@@ -72,11 +73,12 @@ def __init__(
         self.provider = report.check_metadata.Provider
         self.profile = profile
         self.account_id = account
-        self.account_name = organizations.account_details_name
-        self.account_email = organizations.account_details_email
-        self.account_arn = organizations.account_details_arn
-        self.account_org = organizations.account_details_org
-        self.account_tags = organizations.account_details_tags
+        if organizations:
```

Comment:
Why include here the conditional? You are enforcing in the constructor to have a `Organizations_Info` object as argument, empty or not, so the attributes related must be filled in any case, what do you think?